src/frontend/ocamlmerlin/ocamlmerlin.c

```diff
@@ -521,30 +522,84 @@ static void compute_merlinpath(char merlin_path[PATHSZ], const char *argv0, stru
 }
 
 #ifdef _WIN32
+
+/* May return NULL */
+char * retrieve_user_sid_string()
+{
+  char * usidstr = NULL;
+
+  void * process_token = NULL;
```

Comment:
```suggestion
  HANDLE process_token;
```
(don't second-guess Windows API types)